README.md

```diff
@@ -96,7 +96,43 @@ not be repeated, and should be placed in a folder named by date.
 Repeated events are always shown at the top of the events page, and 
 do not expire.
 
-### 4. How do I add a page redirect?
+### 4. How do I add a community document?
+
+A community document is a living document that represents a community effort
+to discuss a question or idea. We call it "living" because we encourage community
+members to edit and contribute to the documents over time, and so there isn't
+a "publish" data associated with it. For this reason, we use [the wiki](https://github.com/USRSE/usrse.github.io/wiki) associated
+with the repository here. 
+
+![assets/img/usrse-book-small.png](assets/img/usrse-book-small.png)
+
+If you ever need to clone this content, you can do:
+
+```bash
+$ git clone git@github.com/USRSE/usrse.github.io/wiki
+```
+
+During the testing phase of the living documents, the wiki will only be editable by community (repository)
+members, however we might open this up to include others as well. 
+
+#### How do I edit an existing document?
+
+To edit a living document that exists, you can browse the [wiki](https://github.com/USRSE/usrse.github.io/wiki) to update it. Generally, we are organizing topics as they are added, so if you want to add a new category or change existing organization, please do so. 
+
+#### How do I start a new document?
+
+If you want to add a new document, while you might add it directly to the wiki, it's recommended to get others
+involved first. You can ping others in the USRSE slack to contribute, and it's also recommended to start the document as a Google
+Document for easier commenting and editing, and then move to the wiki when it's in a final first draft state. 
+For all community documents, the following points apply:
+
+1. It's strongly suggested to first work on your document in Google Docs or similar, where you can bring in multiple community members to put together a first draft. Importantly, you should make sure that the content adheres to the US-RSE [Code of Conduct](https://us-rse.org/code-of-conduct/). At least 3 community members (including yourself) are required before submitting the document to the site. You should feel that it's in a solid "first draft state" before doing a submission.
```

Comment:
It is generally recommended ( and good practice even ) to start a new sentence from a new line in markdown for an easy reviewing process.
Like here `At least 3 community members ...` should be `At least three community members ...`.

Now, due to one line sentence of the entire first point, this review suggestion is coming after the sentence `You should feel ...` which sometimes gives the impression that this comment is for this particular line.

Reply:
That’s definitely something I’ve seen enforced in other communities, but a bit strict so I’m not sure it would fit here, but you could propose it to the rest of the community (Id be fine either way).

Reply:
And I've changed 3 -> three. ;)

Reply:
> That’s definitely something I’ve seen enforced in other communities

Actually, most of the industry content writers especially those who handle Open Source project documentation, and others follow [The IBM Style Guide](https://www.oreilly.com/library/view/the-ibm-style/9780132118989/) and hence probably all follow it. 
But this particular practice of starting a new sentence from a new line does make sense in reviewing process. 😊  

There are many others I found in the whole document but I refrained myself as every project has its own style of documentation. I would, however, propose some layout changes as I get time in the near future (right now least priority for me :) )

Reply:
The presence of a rule in a style guide does not mean that people follow it. If we chose 10 projects, right now at random, most of them would not follow that style.  I can throw up a few random projects, industry standard open source, and show you right now that they do not follow it.

-  Tensorflow: https://raw.githubusercontent.com/tensorflow/tensorflow/master/README.md
-  Nvidia: https://raw.githubusercontent.com/NVIDIA/spark-rapids/branch-0.5/README.md
-  Amazon: https://raw.githubusercontent.com/amzn/style-dictionary/main/README.md

It comes down to wanting a balance of not making it hard for others to contribute, and enforcing rules. Since GitHub added highlighting of single words in a line to show a change, it supports all these communities that don't adhere to one sentence per line. Off the top of my head, I only know of one open source (industry oriented) community that actually follows this rule, and they aren't perfect it at. It's up for debate how much easier it actually makes people's lives, because for every PR (especially for new contributors) they will get the response "Please update to have one sentence per line" and it's more annoying than anything else. A contributor is more likely to have a negative experience because it feels like nit picking things that are somewhat trivial and they will not come back. So, actually, I don't think you are correct that "most industry content writers" follow this rule. My many years of experience across both academic and industry oriented open source projects suggests otherwise.

Also note that this is not an industry open source project, and we try to maximize making it easy to contribute. Thank you for your review!

Reply:
Thanks 😊